scripts/setup-linux.sh

```
@@ -71,34 +71,40 @@ setup_nvm() {
   fi
-
-  if ! bash "$tmp_nvm"; then
-    err "Failed to run nvm installer"
-    return 1
-  fi
-
-  # Source nvm after installation
-  if [ -s "$HOME/.nvm/nvm.sh" ]; then
-    source "$HOME/.nvm/nvm.sh"
-    log "nvm installed and sourced"
+fi
```

Shellcheck (0.11.0)

[error] Expected a '}'. If you have one, try a ; or \n in front of it.

(SC1056)